src/components/pokedex.html

```diff
@@ -87,6 +87,20 @@ <h4 style="margin-bottom: 0">Pokédex</h4>
                                             value="false" onclick="PokedexHelper.updateList()"> Hide alternate forms
                                 </label>
                             </div>
+                            <!--Gender differences-->
+                            <div class="col-12">
+                                <label class="form-check-label">
+                                    <input class="form-check-input" type="checkbox" id="pokedex-filter-gender-diff"
```

Comment:
I can't really see the usecase for this checkbox? Can you explain when you think it's useful?

Reply:
It's just a filter for Pokémon with gender differences, nothing else. I removed it.